sn_cli/src/subcommands/acc_packet.rs

```diff
@@ -14,6 +14,7 @@ use sn_client::registers::EntryHash;
 use sn_client::transfers::HotWallet;
 use sn_client::{Client, FilesApi, FolderEntry, FoldersApi, Metadata, WalletClient};
 
+use crate::subcommands::files;
```

Comment:
you can just put a `self` in below to avoid a separate import

i.e.  `use crate::subcommands::files::{self,  download::download_file, ... };`

Reply:
ok, thanks